### PR TITLE
Added German Translation

### PR DESCRIPTION
**Pull request by GhoulofGSG9**
The original translation was created by ChrisStark. Review and correction was performed by myself.

The translation is following the latest german spelling reform. Words were chosen based on the recommendations of the Dude.

### CODE REVIEW

File: locale/shine/extensions/voterandom/deDE.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"SHUFFLE_AND_RESTART": "{ShuffleType:Translation:voterandom} und starte Runde neu...",
+	"SHUFFLING_TEAMS": "{ShuffleType:Translation:voterandom}...",
+	"TEAMS_SHUFFLED_FOR_DURATION": "{ShuffleType:Translation:voterandom}e Teams wurden aktiviert für die nächsten {Duration:Duration}.",
+	"TEAM_ENFORCING_TIMELIMIT": "{ShuffleType:Translation:voterandom}e Teams erzwingen wurde deaktiviert, da das Zeitlimit erreicht ist.",
+	"TEAM_SWITCH_DENIED": "Du kannst das Team nicht wechseln. {ShuffleType:Translation:voterandom}e Teams sind aktiviert.",
+	"PLACED_ON_RANDOM_TEAM": "Du wurdest in einem zufälliges Team eingeteilt.",
+	"PLACED_ON_HIVE_TEAM": "Du wurdest gemäß deiner Hive Skill Bewertung einem Team zugeteilt.",
+	"ERROR_NOT_ENOUGH_PLAYERS": "Nicht genug Spieler um eine Abstimmung zu starten.",
+	"ERROR_CANNOT_START": "Du kannst derzeit nicht für {ShuffleType:Translation:voterandom}e Teams abstimmen.",
+	"ERROR_ALREADY_ENABLED": "Es wurde bereits für {ShuffleType:Translation:voterandom}e Teams f�r die n�chste Runde abgestimmt.",
```
Comment (Person8880) [inline locale/shine/extensions/voterandom/deDE.json]: I'm guessing it's not supposed to be `f�r die n�chste` here. An encoding issue, or just the wrong characters?

Comment (GhoulofGSG9) [inline locale/shine/extensions/voterandom/deDE.json]: It's an encoding issue.

**Comment by Person8880** [comment]
One other thing, looks like this is open against the wrong branch. :wink:

**Comment by GhoulofGSG9** [comment]
Oh right, I failed to use github again.

**Pull request closed without merging**